app/datasets/[slug]/page.tsx

```diff
@@ -53,42 +53,20 @@ export function generateMetadata({ params }) {
 
 export default function Blog({ params }) {
   let post = getDatasets().find((post) => post.slug === params.slug)
-  console.log(post);
+
   if (!post) {
     notFound()
   }
 
   return (
     <section>
-      <script
-        type="application/ld+json"
-        suppressHydrationWarning
-        dangerouslySetInnerHTML={{
-          __html: JSON.stringify({
-            '@context': 'https://schema.org',
-            '@type': 'BlogPosting',
-            headline: post.metadata.name,
-            datePublished: post.metadata.pubDate,
-            dateModified: post.metadata.pubDate,
-            description: post.metadata.description,
-            url: `${baseUrl}/blog/${post.slug}`,
-            author: {
-              '@type': 'Person',
-              name: 'My Portfolio',
-            },
-          }),
-        }}
-      />
-      <h1 className="title font-semibold text-2xl tracking-tighter">
-        {post.metadata.title}
-      </h1>
-      <div className="flex justify-between items-center mt-2 mb-8 text-sm">
-        <p className="text-sm text-neutral-600 dark:text-neutral-400">
-          {formatDate(post.metadata.publishedAt)}
```

Comment:
These datasets didn't have a `publishedAt`. I think only stories have a `pubDate`?

Reply:
I see. I am not sure `datePublished` is required for Linked Data json, but it might be beneficial to still leave the other fields : https://developers.google.com/search/docs/appearance/structured-data/intro-structured-data